README.md

```diff
@@ -1,5 +1,9 @@
 # Open Source Help Community
 
+ [![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
+
+
```

Comment:
@nikhita  the 3 point you'll got prompt, prompt is not comming and git status shows '
$ git status
Not currently on any branch.
nothing to commit, working directory clean
'